pkg/execution/queue/item.go

```diff
@@ -75,6 +75,20 @@ type Item struct {
 	// QueueName allows control over the queue name.  If not provided, this falls
 	// back to the queue mapping defined on the queue or the workflow ID of the fn.
 	QueueName *string `json:"qn,omitempty"`
+
+	Throttle *Throttle `json:"throttle,omitempty"`
+}
+
+type Throttle struct {
+	// Key is the unique throttling key that's used to group queue items when
+	// processing rate limiting/throttling.
+	Key string `json:"k"`
+	// Limit is the actual rate limit
+	Limit int `json:"l"`
+	// Burst is the busrsable capacity of the rate limit
+	Burst int `json:"b"`
```

Comment:
what exactly is `Burst` in this context?
the additional amount of times we let items through? or is it something else?
if so, should we be doing some accounting on this? like how many bursts were used.

Reply:
It's accounted for in GCRA :)